modules/delete-a-custom-role.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: PROCEDURE
+[id="delete-a-custom-role_{context}"]
+= Deleting a custom role
+
+[role="_abstract"]
+From the *Access Control* view, you can delete the custom roles that you have created.
+
+.Prerequisites
+* You must have *Admin* role, or read and write permissions for the `AuthProvider` and `Role` resources to create, modify, and delete custom roles.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Access control*.
+. Click on the *Roles and Permissions* tab.
+. From the *StackRox Roles* panel, hover over the name of the role you want to delete and click on *Delete icon*.
+
+[NOTE]
+====
+* You cannot modify `read` and `write` permissions for the default system roles.
+* If you have users assigned to a custom role, when you delete that role, all associated users transfer to the configured minimum access role.
```

Comment:
Maybe "If you have users assigned to a custom role and you delete that role,"
or 
"When you delete a custom role that has users assigned to it,"